src/components/movieDetails/movieDetails.tsx

```diff
@@ -0,0 +1,39 @@
+import './movieDetails.css';
+
+interface MovieDetailsProps {
+  imageUrl: string;
+  movieName: string;
+  releaseYear: number;
+  rating: number;
+  duration: number;
+  description: string;
+}
+
+const MovieDetails = (props: MovieDetailsProps) => {
+  const formatDuration = () => {
+    const hours = Math.floor(props.duration / 60);
+    const remainingMinutes = props.duration % 60;
+    return `${hours ? `${hours}h` : ''}${
+      remainingMinutes ? ` ${remainingMinutes}min` : ''
+    }`;
+  };
```

Comment:
Good that you implemented this function.
Points to improve:
 1) provide duration via parameters (formatDuration(duration))
 2) add checking if duration:
    - is a number and a valid number(>0), otherwise use the early return